### PR TITLE
✨ Added extension code for amp-mowplayer

### PR DESCRIPTION
**Pull request by gopanisandip**
Added new extension amp-mowplayer

### CODE REVIEW

**Comment by aghassemi** [comment]
Thanks @gopanisandip!

/to @alanorozco for the first look.

File: gulpfile.js [hunk 1]
```diff
@@ -66,6 +66,105 @@ const unminifiedRuntimeTarget = 'dist/amp.js';
 const unminifiedRuntimeEsmTarget = 'dist/amp-esm.js';
 const unminified3pTarget = 'dist.3p/current/integration.js';
 
+// Each extension and version must be listed individually here.
```
[the remark this answers is not in the document]
Comment (gopanisandip) [reply gulpfile.js]: Could not get this change well. Could you guide me?
I already added,
```javascript
{name: 'amp-mowplayer', version: '0.1', type: TYPES.MEDIA},
```
in bundles.config.js line no 362

Which new code do i need to move? Or i just need to remove line no 69 in gulpfile.js

[the remark this answers is not in the document]
Comment (alanorozco) [reply gulpfile.js]: Remove all `declareExtension` calls from `gulpfile.js`.

What seems to be the problem? :)

[the remark this answers is not in the document]
Comment (gopanisandip) [reply gulpfile.js]: Okay, got it.

Thanks!

[the remark this answers is not in the document]
Comment (alanorozco) [reply gulpfile.js]: Remember to merge from `upstream master` to get the changes to `bundles.config.js` and our overall  build system.

**Comment by gopanisandip** [comment]
Hi @alanorozco 
I recently made some changes suggested by you and marge upstream master to get latest changes. i have one doubt after merge it show commit of other people too.
Is it right or do need to revert this change?

**Comment by aghassemi** [comment]
@gopanisandip There are some options, this document https://gist.github.com/rsimha/a96ec26e00450b9a1477b07c52ecf480 shows how you can fix it 

however my recommendation and probably the safest and easiest is sending another PR. 

You can download a `.diff` of your PR (https://patch-diff.githubusercontent.com/raw/ampproject/amphtml/pull/16697.diff) and then apply it to a new branch and send a new PR based on that branch.

**Comment by aghassemi** [comment]
another set of instructions: https://github.com/ampproject/amphtml/pull/13222#issuecomment-366097749

**Pull request closed without merging**